app/frontend/src/components/forms/SubmissionsTable.vue

```diff
@@ -99,7 +102,7 @@ export default {
           const tableRows = this.submissionList.map((s) => {
             return {
               confirmationId: s.confirmationId,
-              date: s.createdAt,
+              date: this.formatDate(s.createdAt),
```

Comment:
See above. Filters are used everywhere date formatting needs to be to encapsulate that logic. There's a `formatDate` and `formatDateLong` filter there already. Can add additional flags (can pass params to filters) or another filter if a different format is wanted.

Reply:
it did have formatDate, didn't know about those filters.  will set it to formatDateLong.

Reply:
Filters are great, didn't use them on any other Vue apps until now. The existing one uses the long (words) date format, which I think Mina has used in mockups but if we want something else can pass a format to a filter too.